platforms-serverless/netlify-ci/functions/index.js

```diff
@@ -39,10 +39,11 @@ exports.handler = async function (event, context, callback) {
   const tree = dirTree(process.env.LAMBDA_TASK_ROOT);
   console.dir(tree, { depth: null });
   */
- 
+
   // list all files in node_modules/.prisma/client
   const fs = require('fs')
-  const files = fs.readdirSync(process.env.LAMBDA_TASK_ROOT + "/src/node_modules/.prisma/client")
+  const path = require('path')
+  const files = fs.readdirSync(path.dirname(require.resolve('.prisma/client')))
```

Comment:
Why this change? That code works in AWS Lambda generally and is used in other existing tests.

Reply:
Please see https://prisma-company.slack.com/archives/CRGDU19K6/p1623240841039500